admin/spec/service/group.service.spec.js

```diff
@@ -137,6 +193,28 @@ describe('group.service', () => {
       })
     })
 
+    describe('unhappy path', () => {
+      beforeEach(() => {
+        service = proxyquire('../../services/group.service', {
```

Comment:
you shouldnt need proxyquire, it was only needed for mongoose.

Reply:
Removed in parent PR